src/app.rb

```diff
@@ -0,0 +1,138 @@
+require_relative 'library/book'
+require_relative 'library/rental'
+require_relative 'people/school/student'
+require_relative 'people/school/teacher'
+require 'date'
+require 'yaml'
+
+class App
+  MENU = "1 - List all books.
+2 - List all people.
+3 - Create a person.
+4 - Create a book.
+5 - Create a rental.
+6 - List all rentals for a given person id.
+7 - Quit".freeze
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @date = Time.new.strftime('%Y-%m-%d')
+  end
+
+  def self.run
+    print "Select an option(number) to continue.\n"
+
+    loop do
+      clear
+      print "#{MENU}\n"
+      print 'Option: '
+      opt = gets.chomp
+      case opt
+      when '1', '2' then print_list(opt)
+      when '3' then create_person_input
+      when '4' then create_book
+      when '5' then create_rental
+      when '6' then print_rental_by_person_id
+      when '7'
+        clear
+        break
+      end
+      puts ''
+      puts 'Press enter to continue'
+      gets.chomp
+    end
+  end
+
+  def self.create_book
+    puts 'Input title and author'
+    print 'Title: '
+    title = gets.chomp
+    puts ' '
+    print 'Author: '
+    author = gets.chomp
+    current = @@books.length
+    create_book(title, author)
+    puts current < @@books.length ? 'Book added' : 'Error'
+    @@books.push(Book.new(title, author))

```

Comment:
- [ ] I've got the following error while I'm trying to create a new book, please double-check the implementation of your method

`uninitialized class variable @@book in App (NameError)`